wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -181,13 +183,17 @@ pub enum WalletCommand {
     SyncWallet,
 
     GetBalance {
+        #[arg(value_enum, default_value_t = CliWithLocked::Unlocked)]
```

Comment:
Up for a discussion but I'd show all utxos and balances by default. For balance it could be a separate "locked balance" line.

Reply:
I prefer to show only spendable coins. We can add a note that this displays only spendable coins, and they have to check `help getbalance` for more information.